expression/integration_test.go

```diff
@@ -6620,6 +6620,28 @@ func (s *testIntegrationSuite) TestIssue17287(c *C) {
 	tk.MustQuery("execute stmt7 using @val2;").Check(testkit.Rows("1589873946"))
 }
 
+func (s *testIntegrationSuite) TestIssue17727(c *C) {
```

Comment:
could you also add a case to test the parameter on `unix_timestamp`? for example:
```sql
prepare stmt from 'select unix_timestamp(?)';

set @a = '2020-05-30 20:30:00';
execute stmt using @a;
select @@last_plan_from_cache; -- should be false

set @a = '2020-06-12 13:47:58';
execute stmt using @a; -- should get a different result
select @@last_plan_from_cache; -- should be true

```

Reply:
@zz-jason Update. PTAL